04-shapes/Shapes.java

```diff
@@ -0,0 +1,137 @@
+        r = input.nextDouble();
+        circle(r);
+      }
+      if (formula == 5){//    CUBE
+        System.out.println("Enter the side length: ");
+        s = input.nextDouble();
+        cube(s);
+    }
+      if (formula == 6){//    TRIANGULAR PRISM
+        System.out.println("Enter the base: ");
+        b = input.nextDouble();
+        System.out.println("Enter the Height: ");
+        h = input.nextDouble();
+        System.out.println("Enter the length: ");
+        l = input.nextDouble();
+        triprism(b,l,h);
+      }
+      if (formula == 7){//    RECTANGULAR PRISM
+        System.out.println("Enter the Length: ");
+        l = input.nextDouble();
+        System.out.println("Enter the Width: ");
+        w = input.nextDouble();
+        System.out.println("Enter the Height: ");
+        h = input.nextDouble();
+        rectprism(l,w,h);
+      }
+      if (formula == 8){
+        System.out.println("Enter the radius: ");
+        s = input.nextDouble();
+        sphere(s);
+      }
+      System.out.println("\n1)calculator \n2)Exit");
+      int Cal = input.nextInt();
+      if (Cal == 2){//start if statement
+        break;
+      }//end if statement
+    }
+}//end class main
+  public static void square(double s){//SQUARE
+    double a = s * s;
+    System.out.println("A square with side " + s + " has an area of " + a);
+    writeChars('-',11);
+    System.out.println("");
+      for (int i = 0;i<=4;i++){
+      writeChars('-',1);
+      writeChars(' ',9);
+      writeChars('-',1);
+      System.out.println("");
+   }
+    writeChars('-',11);
+  }//close square method
+  public static void triangle(double b,double h){//TRIANLGE
+    double a = (1.0/2*(b*h));
+    System.out.println("A triangle with a base of " + b + " and the height of " + h + " has an area of " + a);
+    for (int i = 10; i >= 1; i--) {
+      writeChars('>', i);
+      writeChars(' ', 20 - 2 * i);
+      writeChars('<', i);
+      System.out.println();
+    }
+    writeChars('=', 20);
+  }//close triangle method
+  public static void rectangle(double l, double w){//RECTANGLE
+    double a = (l*w);
+    System.out.println("A rectangle with a length of " + l + " and the width of " + w + " has an area of " + a);
+    writeChars('-',20);
+    System.out.println("");
+      for (int i = 0;i<=4;i++){
+      writeChars('-',1);
+      writeChars(' ',18);
+      writeChars('-',1);
+      System.out.println("");
+   }
+    writeChars('-',20);
+  }//close rectangle method
+  public static void circle(double r){//CIRCLE
+    double cir = (3.14*(r*r));
+    System.out.println("A circle with a radius of " + r + " has an area of " + cir);
+  }//close circle method
+  public static void cube(double s){//CUBE
+    double v = (s*s*s);
+    System.out.println("A cube with a side of " + s + " has an volume of " + v);
+  }//close circle method
+  public static void triprism(double b,double l, double h){//TRIANGULAR PRISM
+    double v = (1.0/2*(b*l*h));
```

Comment:
couldn't you reuse the code from `triangle()` since the exact formula shows up here?